index.slim

```diff
@@ -0,0 +1,56 @@
+---
+---
+doctype html
+html lang="en"
+  head
+    meta charset="utf-8" /
+    meta content="IE=edge" http-equiv="X-UA-Compatible" /
+    meta content="width=device-width, initial-scale=1" name="viewport" /
+    link href="/img/fav/apple-touch-icon.png" rel="apple-touch-icon" sizes="180x180" /
+    link href="/img/fav/favicon-32x32.png" rel="icon" sizes="32x32" type="image/png" /
+    link href="/img/fav/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png" /
+    link href="/img/fav/site.webmanifest" rel="manifest" /
+    link color="#3241a6" href="/img/fav/safari-pinned-tab.svg" rel="mask-icon" /
+    link href="/img/fav/favicon.ico" rel=("shortcut icon") /
+    meta content="#3241a6" name="msapplication-TileColor" /
+    meta content="/img/fav/browserconfig.xml" name="msapplication-config" /
+    meta content="#3241a6" name="theme-color" /
+    link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" type="text/css" /
+    link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" /
```

Comment:
What is your opinion on instead of using this CDN we package this in the repo?

Reply:
I think we would have a bigger repo which would lead to longer building/deploying times. It would also make it less convenient to upgrade to new versions.

They are trade-offs. I like using their CDN because is a recomended way from their documentation and it is a reliable one.

Reply:
What's yours? Have they ever failed you?

Reply:
Webpack maybe interesting in looking at. We wouldn't need more repo "space" and build times shouldn't increase because of caching.
With this we would be in more control over our delivery of assets.

Reply:
How would a do it? :thinking: Like adding a gem for this? Or npm package?

Any useful link for reference?

Reply:
A simple starting point could be [How to Use ES6 and Webpack With Jekyll](https://michaelmovsesov.com/articles/jekyll-es6-workflow)

Reply:
I really don't want to do it right now. I opened an issue (#2) for future reference and give the opportunity to anyone to start that work.